src/main/treeDataProviders/dependenciesTree/dependenciesDataProvider.ts

```diff
@@ -105,53 +105,35 @@ export class DependenciesTreeDataProvider implements vscode.TreeDataProvider<Dep
 
     private async scanAndCacheComponents(progress: vscode.Progress<{ message?: string; increment?: number }>, checkCanceled: () => void) {
         const totalComponents: number = this._componentsToScan.size() + this._goCenterComponentsToScan.size() || 1;
-        progress.report({ message: 0 + '/' + totalComponents + ' components scanned' });
+        progress.report({ message: `Scanning ${totalComponents} components` });
```

Comment:
This is not true. Please leave it as it was.

Reply:
We use `scanAndCacheComponents` two times, one time with Go Center and the second time with XRay.
As a result, one report will override the other and we will never see to 100%.

Reply:
Let's talk about it tomorrow